### PR TITLE
adds app.js and reviews.jsx

### PR DESCRIPTION
**Pull request by Vondry66**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -11,7 +11,7 @@ class App extends Component {
         <Header />
         <Nav />
         <Routes>
-          <Route path="Reviews" element={<Reviews />}></Route>
+          <Route path="Reviews" element={<Reviews />}>r</Route>
```
Comment (leekli) [inline src/App.js:14]: Is this 'r' here by accident?

File: src/components/Reviews.jsx [hunk 2]
```diff
@@ -17,7 +17,7 @@ const Reviews = ()=>{
                     return <li key={review.review_id}>
                         <h3>{review.title}</h3>
                         <p>{review.category}</p>
-                        <p>{review.body}</p>
+                        <p>{review.body}r</p>
```
Comment (leekli) [inline src/components/Reviews.jsx:20]: Is this 'r' here by accident?

**Comment by Vondry66** [comment]
the "r" is there just so i could make changes and actually push the branch

**Review by leekli — APPROVED** [review]
Hi Filip, good to merge however...

* There are to rogue 'r's in your code, see my comments
* Also on line 9 in Reviews.jsx there's a console log which does not need to be there

File: src/App.js (hunk 1 above)
Comment (Vondry66) [reply src/App.js:14]: the "r" is there just so i could make changes and actually push the branch. Thank you, fixed.

**Pull request merged**